### ISSUE TITLE
[Enh] More details in running status

### ISSUE THREAD

**Issue by JusteRaimbault** [dafnifacility/user-feedback#41]
(related to [#15](https://github.com/dafnifacility/user-feedback/issues/15) and  [#5](https://github.com/dafnifacility/user-feedback/issues/5) )
It would be useful to have more info on the running status of a workflow: which task (or tasks if an iterator is run in parallel) is currently running; how many cores are currently being used.

**Comment by BethanPerkins** [comment]
Thanks for this Juste. Could you say a little more about how this will help you - are you essentially looking for an estimate of time remaining until the workflow is complete?

**Comment by JusteRaimbault** [comment]
Yes that was basically the idea - for a workflow with multiple tasks, seeing the current task running (which we can already see on the workflow page, but a summary on the workflow list may be useful); and for workflows with multiple tasks run in parallel (an iterator for example), see how much tasks have finished, how much are currently running, and how much are remaining (and maybe an estimate of the time needed, based on previous model executions within the iterator?).

**Comment by ajhickford** [comment]
On a related issue, is it possible to set up an email alert or similar to let a user know when a model run has completed?

**Comment by JusteRaimbault** [comment]
And also a related idea (putting it here to avoid opening a new issue) - could it be possible to display info about the memory, such as how much memory the model currently running is using, how much it has available, etc. ? (for models requiring large amounts this may be useful to check it does not fail because of this)

**Comment by tomalrussell** [comment]
Piling ideas on here - a view into STDOUT/STDERR logs would be great too, for monitoring progress, sensechecking and debugging.

**Comment by BethanPerkins** [comment]
Lots of ideas to capture here, thanks for the detail. I've got the following: 
- Provide user with some runtime information from a workflow, indicating wherever possible the progress of the model. This could be as simple as STDOUT/STDERR to begin with and could include memory usage. (DAFNI internal ticket [CORE-1462](https://jira.dafni.rl.ac.uk/browse/CORE-1462))
- Include option to send a notification email when a workflow has completed. (DAFNI internal ticket [CORE-1461](https://jira.dafni.rl.ac.uk/browse/CORE-1461))

**Comment by JusteRaimbault** [comment]
An additional feature useful would be to see how much time and CPU equivalent time a workflow execution (and its subtasks) took for a finished workflow (for now there is only the start time displayed if I'm not mistaken?)

**Comment by fmcclean** [comment]
> An additional feature useful would be to see how much time and CPU equivalent time a workflow execution (and its subtasks) took for a finished workflow (for now there is only the start time displayed if I'm not mistaken?)

Linked to #11
